docs/src/index.md

```diff
@@ -6,6 +6,48 @@ CurrentModule = AlignedSpans
 
 See [API documentation](@ref) for how to construct AlignedSpans, along with some utilities, or below for some examples and motivation.
 
+### Continuous -> Discrete
+
+Continuous timespans can be rounded (or "aligned") to the individual sample values by using the constructor `AlignedSpan`, which takes a `sample_rate`, a `span`, and a description of how to round time endpoints to indices. This constructs an `AlignedSpan` which supports Onda indexing. Internally, an `AlignedSpan` stores indices, not times, and any rounding happens when it is created instead of when indexing into `samples`.
+
+Rounding options:
+
+* `EndpointRoundingMode`: consists of a `RoundingMode` for the `start` and `stop` of the span.
+    * The alias `RoundInward = EndpointRoundingMode(RoundUp, RoundDown)`, for example, constructs the largest span (whose endpoints are valid indices) that is entirely contained within `span`.
+    * The alias `RoundEndsDown = EndpointRoundingMode(RoundDown, RoundDown)` matches the rounding semantics of `TimeSpans.index_from_time(sample_rate, span)`.
```

Comment:
```suggestion
    * The alias `RoundDown = EndpointRoundingMode(RoundDown, RoundDown)` matches the rounding semantics of `TimeSpans.index_from_time(sample_rate, span)`.
```

Reply:
When reading RoundEndsDown I expected that only the end time is rounded down. Maybe this alias is more intuitive.

Reply:
Not shure if there is now a symbol clash as `RoundDown` was existing before?